src/micro_http/src/response.rs

```diff
@@ -98,6 +105,20 @@ impl ResponseHeaders {
         buf.write_all(b"Connection: keep-alive")?;
         buf.write_all(&[CR, LF])?;
 
+        if !self.allow.is_empty() {
+            buf.write_all(b"Allow: ")?;
+
+            let delimitator = b", ";
+            for (idx, method) in self.allow.iter().enumerate() {
+                buf.write_all(method.raw())?;
+                if idx < self.allow.len() - 1 {
+                    buf.write_all(delimitator)?;
+                }
+            }
+
+            buf.write_all(&[CR, LF])?;
+        }
+
```

Comment:
nit: Keep this fn small and readable - move this code inside a `write_allow()` or something else.